File: Python/factorial-trailing-zeroes.py
import math
class Solution(object):
    def trailingZeroes(self, n):
        """
        :type n: int
        :rtype: int
        """
        sum = 0
        i = 5
        while i <= n:
            sum += n // i
            i *= 5
        return sum

print(Solution().trailingZeroes(6503))

File: Golang/factorial-trailing-zeroes.go
package main

import "fmt"

func trailingZeroes(n int) int {
	sum := 0
	i := 5
	for i <= n {
		sum += n / i
		i *= 5
	}
	return sum
}

func main() {
	fmt.Println(trailingZeroes(1674))
}
